webplugin/js/app/components/answer-feedback-service.js

```
@@ -150,7 +150,9 @@ class AnswerFeedback {
 
         if (
             currentUser.role !==
-            KommunicateConstants.GROUP_ROLE.MODERATOR_OR_BOT
+                KommunicateConstants.GROUP_ROLE.MODERATOR_OR_BOT ||
+            msg.metadata.hasOwnProperty('KM_ANSWER_FEEDBACK') ||
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)